packages/tools/client-debugger/client-debugger-view/src/components/ContainerView.tsx

```diff
@@ -2,9 +2,10 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-import { IOverflowSetItemProps, IconButton, Link, OverflowSet, Stack } from "@fluentui/react";
+import { Stack } from "@fluentui/react";
+import { Tab, TabList, SelectTabData, SelectTabEvent, TabValue } from "@fluentui/react-components";
 import { HasContainerId } from "@fluid-tools/client-debugger";
-import React from "react";
+import * as React from "react";
```

Comment:
It seems we could stick with `import React from "react"` and things would still work, right? The PR only uses it for `React.useState()` which works the same in other files with the previous version of the import. Let's try to keep it consistent.